teps/0098-workflows.md

```diff
@@ -412,16 +434,236 @@ and Pipeline definitions into a higher level opinionated syntax that is aware of
 
 ## Proposal
 
+Initially, the cluster operator will install Tekton Workflows onto their Kubernetes cluster. They will connect an individual repo
```

Comment:
you are mentioning a cluster operator operation, I can understand being a cluster operation for github apps (since it handles multiple repos) but why does it need to be a cluster operation for webhook ? does it need to be a cluster-admin operation?

Reply:
Ah this is a good point; would it be more clear if we say:
- Cluster operator installs tekton workflows
- Repo admin connects a repo to tekton workflows (I'm not sure exactly what cluster permissions they'd need here: maybe permissions to update configmaps in the installation namespace? or permissions to create a Repo CRD in any namespace? or maybe workflows looks up the repo admin, and having repo admin permissions is sufficient)
- If you want to connect a GitHub app, which can be installed on multiple repos in an org, I think we'd probably want you to be an org admin. Again not sure what cluster permissions should be required-- it probably depends on how a "connection" is implemented (stored in a CRD, in a configmap, ???)

@alhuizenga thoughts?

Reply:
I reworded to "repo admins" and "organization admins"; I don't think you should necessarily have to be a cluster admin to connect a repo